README.md

```diff
@@ -37,42 +53,32 @@
-* [Networked servers](https://goteleport.com/docs/server-access/introduction/)
+We have implemented Teleport as a single Golang binary that integrates with multiple protocols and cloud services:
 
-Teleport is trivial to set up as a Linux daemon or in a Kubernetes pod. It's rapidly
-replacing legacy `sshd`-based setups at organizations who need:
+* [SSH nodes](https://goteleport.com/docs/getting-started/).
+* [Kubernetes clusters](https://goteleport.com/docs/kubernetes-access/introduction/)
+* [PostgreSQL, MongoDB, CockroachDB and MySQL databases](https://goteleport.com/docs/database-access/introduction/).
+* [Internal Web apps](https://goteleport.com/docs/application-access/introduction/).
+* [Windows Hosts](https://goteleport.com/docs/desktop-access/introduction/).
+* [Networked servers](https://goteleport.com/docs/server-access/introduction/).
 
-* Developer convenience of having instant secure access to everything they need
-  across many environments and cloud providers.
-* Audit log with session recording/replay for multiple protocols
-* Easily manage trust between teams, organizations and data centers.
-* Role-based access control (RBAC) and flexible access workflows (one-time [access requests](https://goteleport.com/features/access-requests/))
+You can set up Teleport as a [Linux daemon](https://goteleport.com/docs/#set-up-a-demo-cluster) or a [Kubernetes deployment](https://goteleport.com/docs/deploy-a-cluster/helm-deployments/).
 
-In addition to its hallmark features, Teleport is interesting for smaller teams
-because it facilitates easy adoption of the best infrastructure security
-practices like:
+Teleport focuses on the best infrastructure security practices:
 
-- No need to manage shared secrets such as SSH keys: Teleport uses certificate-based access with automatic certificate expiration time for all protocols.
+- No need to manage shared secrets such as SSH keys or Kubernetes tokens: it uses certificate-based auth with certificate expiration for all protocols.
 - Two-factor authentication (2FA) for everything.
 - Collaboratively troubleshoot issues through session sharing.
-- Single sign-on (SSO) for everything via GitHub Auth, OpenID Connect, or SAML with endpoints like Okta or Active Directory.
+- Single sign-on (SSO) for everything via GitHub Auth, OpenID Connect, or SAML with endpoints like Okta or Active Directory [2]
 - Infrastructure introspection: Use Teleport via the CLI or Web UI to view the status of every SSH node, database instance, Kubernetes cluster, or internal web app.
 
-Teleport is built upon the high-quality [Golang SSH](https://godoc.org/golang.org/x/crypto/ssh)
-implementation. It is _fully compatible with OpenSSH_,
-`sshd` servers, and `ssh` clients.
+[2] Open-Source version only supports Github SSO.
+
+Teleport uses [Golang crypto](https://godoc.org/golang.org/x/crypto). It is _fully compatible with OpenSSH_, `sshd` servers, and `ssh` clients, Kubernetes clusters and more.
```

Comment:
```suggestion
Teleport uses [Go crypto](https://godoc.org/golang.org/x/crypto). It is _fully compatible with OpenSSH_, `sshd` servers, and `ssh` clients, Kubernetes clusters and more.
```

Reply:
I would also say "it is compatible with OpenSSH" - not "fully compatible".

"fully" is just a filler word in this case - the sentence reads just the same without it, and then you don't open up debate as to what "fully" actually means.